Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtension.mm

```diff
@@ -199,6 +199,12 @@
     EXPECT_NS_EQUAL(testExtension.errors, @[ ]);
     EXPECT_NS_EQUAL(testExtension.displayActionLabel, @"Button Title");
     EXPECT_NOT_NULL([testExtension actionIconForSize:NSMakeSize(16, 16)]);
+
+    testManifestDictionary = @{ @"manifest_version": @3, @"name": @"Test", @"description": @"Test", @"version": @"1.0", @"action": @{ } };
+    testExtension = [[_WKWebExtension alloc] _initWithManifestDictionary:testManifestDictionary];
+    EXPECT_NS_EQUAL(testExtension.errors, @[ ]);
```

Comment:
Are there any other assertions that we should add, or is having an empty action in the manifest and checking that there aren't any errors sufficient?

Reply:
That looks good.